ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -1050,6 +1083,7 @@ namespace dpct
                     _cpu_device = _devs.size() - 1;
                 }
             }
+	    init_ccl();
```

Comment:
1. mv this init() function to ggml-sycl.cpp/h.
2. please consider the case of no tensor parallel. It's more common.
User needn't install oneCCL and MPI in this case.